apps/angular/src/core/utils/equal-passwords-validator.ts

```
@@ -0,0 +1,22 @@
+import { AbstractControl, ValidationErrors, ValidatorFn } from '@angular/forms';
+
+/**
+ * Checks whether two strings in a form are equal.
+ * @param controlOne First field.
+ * @param controlTwo Second field.
+ * @returns
+ */
+export function equalityValidator(controlOne: string, controlTwo: string): ValidatorFn {
+	return (control: AbstractControl): ValidationErrors | null => {
+		const valueOne = control.get(controlOne);
+		const valueTwo = control.get(controlTwo);
+
+		if (valueOne !== null && valueTwo !== null && valueOne.value !== valueTwo.value) {
```

I think if `valueOne` or `valueTwo` is missing we should throw an error because it's unexpected behavior